src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -359,6 +366,17 @@ public static PreferenceSettings Load(string filePath)
             settings.CustomPackageFolders = settings.CustomPackageFolders.Distinct().ToList();
 
             return settings;
+		}
+
+        private void InitializeNamespacesToExcludeFromLibrary()
```

Comment:
can we call this function from the PreferenceSettings ctor? If so then we don't have to rely on the settings file to exist in order for the namespace list to  be initialized